dpnp/dpnp_iface_histograms.py

```diff
@@ -45,6 +45,12 @@
 
 import dpnp
 
+# pylint: disable=no-name-in-module
```

Comment:
It looks working for me without the muted rule
```suggestion
```

Reply:
It is working without muted rule in case module already being built.

But on CI pre-commit runs before build.